### DISCUSSION TITLE
New to contribution.

### DISCUSSION THREAD

**Discussion by Ruchit1101** [koordinates/kart#763] [Google Soc / SoD]
I am new to open source and want to contribute to the projects. So, I need the guidance for the same.

**Comment by rcoup** [comment]
Hi, we haven't decided whether to participate in GSoC 2023 yet... (and if we do, whether we'll be accepted) — check back at the end of February

**Reply by Ruchit1101** [reply]
Thank you. But still I want to contribute to this organization, so can you please guide me to the first step ?
